multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/BindServiceToApplicationStep.java

```diff
@@ -22,6 +23,8 @@
 @Scope(BeanDefinition.SCOPE_PROTOTYPE)
 public class BindServiceToApplicationStep extends SyncFlowableStep {
 
+    private static final int CF_SERVICE_ALREADY_BOUND = 90003;
```

Comment:
What is this code? I think it's not realiable to use such custom code to determine already bound service to application.

Reply:
It is coming from CC and it is used in CF CLI:
https://github.com/cloudfoundry/cloud_controller_ng/blob/main/errors/v2.yml#L391